apps/server/src/services/storageClient/redis.ts

```diff
@@ -1,86 +1,88 @@
-import { readFileSync } from 'fs'
-import { resolve } from 'path'
-import { createClient } from 'redis'
-import config from '../../config'
+import { readFileSync } from "fs";
+import { resolve } from "path";
+import { createClient } from "redis";
+import config from "../../config";
 
-import { debug, error, info } from '../../infra/logger'
-import { PREFERENCES_REDIS_KEY } from './constants'
+import { debug, error, info } from "../../infra/logger";
+import { PREFERENCES_REDIS_KEY } from "./constants";
+
+export * as REDIS_CONSTANTS from "./constants";
```

Comment:
This export is necessary to expose constant that mx adapter needs.

Reply:
Other changes in this file are from Prettier.

Reply:
Why doesn't whatever needs it import it from the constants file?

Reply:
The new mx adapter package needs it. It's being passed in as a dependency in `adapterSetup.ts` in the forked repo.

Reply:
After looking this over, I actually don't need this constant. The entire redis cache instance is being passed in. I'll test it without this export to make sure it works. If so, I'll push up a new PR with this reverted.